models/urdfs/panda/panda.urdf

```diff
@@ -0,0 +1,321 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!-- Manually converted from franka_description/robots/panda_gazebo.xacro -->
```

Comment:
Here's the permalink, just for reference/posterity:

https://github.com/frankaemika/franka_ros/blob/8dc80d19aeb2aa88625b1ad42e03d8e8169e3905/franka_description/robots/panda_gazebo.xacro

(correct me if this is _not_ the file this was converted from)

Reply:
Correct, but I think this comment is better than a permalink